src/include/commands/matview.h

```diff
@@ -34,4 +34,6 @@ extern bool MatViewIncrementalMaintenanceIsEnabled(void);
 
 extern Datum IVM_immediate_maintenance(PG_FUNCTION_ARGS);
 
+extern Query *get_materializedview_query(Relation matviewRel, bool concurrent);
```

Comment:
This can be static function instead of extern because this will be called only in matview.c.

Reply:
I will fix too.

Reply:
Are you not going to use this function at other place(for example createas.c) In the feature?

Reply:
I am not exactly sure , but I don't think so, especially for createas.c, because when creating a materialized view the query information is available from the parsed statement. At least for now, this is not called from other place, so I think using static is better for good modularity. If we come to find the need to call this from outer, we can change this to use extern at that time.

Reply:
Although we might consider this when implementing ALTER MATERIALIZED VIEW if this is an appropriate manner....

Reply:
Ok,  If it will be need by other codes, we consider at that time.